akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -347,6 +347,18 @@ class Http2ServerSpec extends AkkaSpec("""
         sendRST_STREAM(TheStreamId, ErrorCode.CANCEL)
         entityDataOut.expectCancellation()
       }
+
+      "handle RST_STREAM while data is in buffer" in new WaitingForResponseDataSetup {
```

Comment:
I found it hard to describe in a clear way the exact scenario is. Ideas?

Reply:
Sounds pretty good, maybe "while data is waiting in outgoing stream buffer"